src/worker/startup.ts

```diff
@@ -12,6 +12,8 @@ export async function start() {
 		return;
 	}
 
+	await
```

Comment:
Whats this `await` for? Is this something that got missed out?

Reply:
I think this is causing the 500 Internal server error in the test case. Can you please revert this part?

Reply:
woops, good catch, it was something I was trying out.